install_requirements.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# Copyright 2024-25 Arm Limited and/or its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import sys
+
+import install_executorch
+
+
```

Comment:
np: don't you wanna do the other way around? 

move the `install_requirements()` function into this file instead.

Reply:
done